src/control/cmd/daos_server/main.go

```diff
@@ -152,7 +152,9 @@ func parseOpts(args []string, opts *mainOpts, log *logging.LeveledLogger) error
 			if err := cfgCmd.loadConfig(opts.ConfigPath); err != nil {
 				return errors.Wrapf(err, "failed to load config from %s", cfgCmd.configPath())
 			}
-			log.Infof("DAOS Server config loaded from %s", cfgCmd.configPath())
+			if _, err := os.Stat(opts.ConfigPath); err == nil {
```

Comment:
I was expecting to have this information message in your new function `func (c *optCfgCmd) loadConfig(cfgPath string) error`.

Reply:
Then we need to print the information both in "func (c *cfgCmd) loadConfig(cfgPath string)" and "func (c *optCfgCmd) loadConfig(cfgPath string)". Do you prefer this way?

Reply:
Do not bother too much with that, I am not a GO expert and thus I am not sure if that is the best coding practice.